utils/test_memcheck.supp

```diff
@@ -403,19 +399,18 @@
    Memcheck:Leak
    ...
    fun:calloc
+   ...
    fun:vea_reserve
-   fun:ut_*
-   obj:/usr/lib64/libcmocka.so*
-   fun:_cmocka_run_group_tests
-   fun:main
+   ...
```

Comment:
This one also seems problematic

Reply:
Yes, you're right, this can cover real issues calling vea_reserve. Let me investigate this one further.